src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCodeV1.java

```diff
@@ -47,6 +47,8 @@ public enum ErrorCodeV1 {
   /** note: Only used by EmbeddingGateway */
   INVALID_REQUEST("Request not supported by the data store"),
 
+  INVALID_REQUEST_STRUCTURE_MISMATCH("Request invalid, mismatching JSON structure"),
```

Comment:
@vkarpov15 Changed error code, wording, is this any closer to being acceptable? I am happy to keep on tweaking it until we get it right.

Reply:
I think this message is good enough, thanks for listening :+1: